README.md

```diff
@@ -10,7 +10,7 @@ To see test results, click the status badge for the relevant package in the list
 
 <!-- START:PACKAGES -->
 
-| Package                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                                        Latest Stylelint |                                                                                                                                                                                                                                                                                                                    Next Stylelint |
+| Package                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                               Latest Stylelint (16.7.0) |                                                                                                                                                                                                                                                                                                                    Next Stylelint |
```

Comment:
Should we maybe als use an npm badge here?
So that it is always up to date?

Reply:
Ah, nice idea! I'll update it to use badge 👍🏼

Reply:
There's a slight shift between the image and the text, but do you have any idea?

<img width="240" alt="image" src="https://github.com/user-attachments/assets/91955d0d-12c9-4e77-9c06-4d4b2c299c43">

Reply:
I found the `label` query option:

```markdown
[![Latest Stylelint](https://img.shields.io/npm/v/stylelint.svg?label=Latest+Stylelint)](https://www.npmjs.com/package/stylelint)
```

How about this?

[![Latest Stylelint](https://img.shields.io/npm/v/stylelint.svg?label=Latest+Stylelint)](https://www.npmjs.com/package/stylelint)

Reply:
> There's a slight shift between the image and the text, but do you have any idea?
> 
> <img alt="image" width="240" src="https://private-user-images.githubusercontent.com/473530/352859139-91955d0d-12c9-4e77-9c06-4d4b2c299c43.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyMjAwMTgsIm5iZiI6MTcyMjIxOTcxOCwicGF0aCI6Ii80NzM1MzAvMzUyODU5MTM5LTkxOTU1ZDBkLTEyYzktNGU3Ny05YzA2LTRkNGIyYzI5OWM0My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOVQwMjIxNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNzIwNDkxZDg4ZDczMzMyNzFkZGJlYmE3NGI2ZTQ5YmVlYmMzMTU2ZTYzZGYxNDBjZjFkZDAwYmI2NWUxZWY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.vimnlXK6uRWljsippLM9EhvCNG4hT0nu6PIaQ1TqWFM">

Try adding `align="middle"`.

Reply:
Isn't `align` deprecated?

Reply:
All browsers support it and `style="…"` would be ignored by GitHub so you don't have much choice.

Reply:
How do we set `align` in Markdown?

Reply:
```html
<a href="https://www.npmjs.com/package/stylelint">Latest Stylelint</a> <img align="top" alt="Latest Stylelint" src="https://img.shields.io/npm/v/stylelint.svg" />
```

<a href="https://www.npmjs.com/package/stylelint">Latest Stylelint</a> <img align="top" alt="Latest Stylelint" src="https://img.shields.io/npm/v/stylelint.svg" />

Reply:
Markdown also seems ok to me :)
We can improve as we go along.